dev-packages/scripts/package.json

```diff
@@ -0,0 +1,30 @@
+{
+    "private": true,
+    "name": "@eclipse-che/scripts",
```

Comment:
it looks like too generic. It's for 'che-theia' there no ? (even if it's unpublished/private)
`@eclipse-che/theia-scripts`

Reply:
OK